### PR TITLE
deploy: URL defaulting to http://localhost:8701

### PR DESCRIPTION
**Pull request by sam-github**
Connect to strongloop-internal/scrum-nodeops#289

### CODE REVIEW

**Comment by sam-github** [comment]
This should be working and ready to merge... its backwards compatible.

File: bin/sl-deploy.js [hunk 1]
```diff
@@ -62,8 +64,14 @@ var baseURL = argv[parser.optind()];
 if (numArgs === 2) {
   branchOrPack = argv[parser.optind() + 1];
 }
+baseURL = baseURL || 'http://';
 branchOrPack = branchOrPack || 'deploy';
 
+// Truncate any paths from the baseURL, because the git push does a raw string
+// concatenation of '/<config>', and older versions of deploy allowed paths on
+// the git push.
+baseURL = defaults(baseURL, {host: 'localhost', port: 8701}, {path: '/'});
```
Comment (kraman) [inline bin/sl-deploy.js]: Might want to default to 127.0.0.1 instead of localhost. Depending on the OS, localhost may resolve to ::1 instead.

**Comment by kraman** [comment]
LGTM other than the localhost vs 127.0.0.1 issue.

**Comment by sam-github** [comment]
OK, thanks, its now 127.0.0.1.

**Pull request merged**